src/test/java/org/alfresco/repo/event2/UpdateRepoEventIT.java

```diff
@@ -497,7 +498,6 @@ public void testUpdateNodeTypeWithCustomType()
 
         NodeResource resourceBefore = getNodeResourceBefore(3);
         assertEquals("Incorrect node type was found", "cm:content", resourceBefore.getNodeType());
-        // assertNotNull(resourceBefore.getModifiedAt()); uncomment this when the issue will be fixed
```

Comment:
maybe add a TODO so it doesn't get missed?

Reply:
If you remember we closed the raised issue as "Not a Bug", so there is no need for the commented code or TODO.